### ISSUE TITLE
Fine tuning time

### ISSUE THREAD

**Issue by Polarbear2121** [IIEleven11/StyleTTS2FineTune#4]
You mentioned that you used an A600 for 1 hour dataset and it took about 3.5 to 4 days.  Could you comment on your results?  If so, which metric "loss" did you use to stop the training?  How many epochs?

**Comment by IIEleven11** [comment]
Uhm so I actually just hit a great checkpoint. Its not done yet but its an almost perfect result. What you're hearing is the 39th epoch of about an hour long dataset with my improved segmentation and transcription scripts. 
[EM15E_12_6_23_0326_11.webm](https://github.com/IIEleven11/StyleTTS2FineTune/assets/55998857/1ddc3050-270d-4b69-a5ac-4327c55629f3)
As for the loss metrics, what I was doing was waiting for them to level out, attempting not to overfit. But this last run i've actually been taking screenshots of my tensorboards and letting Google Bard analyze them for me along with a snippet of either the training logs and or the config file. That way it can suggest things worth looking into. Pair that with what you know about identifying training progress and I think it turns into a great tool, given how complicated all of this can get

**Comment by Polarbear2121** [comment]
How many days in training?  Did your A600 handle a max_len of 600 and a batch size of 2, or did you use another GPU?  By the way, great job!!!

**Comment by IIEleven11** [comment]
> How many days in training?  Did your A600 handle a max_len of 600 and a batch size of 2, or did you use another GPU?  By the way, great job!!!

Oh sorry, that took about 12 hours. Even with the accelerated fine tuning it goes so so slow. If the model is working well with your dataset it should in theory get faster and faster as time goes on. Right now it shows it has gone from a training speed of 468 to about 680.
